db/seeds.rb

```diff
@@ -1,76 +1,38 @@
-# Cases we should cover with seeds
-# - User with no resources, no lists, no groups
-# - User with no resources, one list, no groups
-# - User with resource, list, group
-# - User with one group, no list
-# - Group with all users, one list
-# - Group with 2 users, no list
-# - Resource belongs to a user, and two lists
-# - Resource belongs to no user, and two lists
-# - Resource belongs to one list
-# - User's list
-# - User's list
-# - Group's list
-
-# Create users
-5.times do |n|
-  User.create(
-    email: "user#{n}@greencommons.org",
-    password: 'thecommons'
-  )
+# config
+users_count = 20
+resources_count = 400
+groups_count = 10
+
+p "Creating #{users_count} users..."
+users_count.times do |n|
+  FactoryGirl.create(:user, email: "user#{n}@greencommons.org",
```

Comment:
My only concern with this is that it doesn't give a user for the developer to play with. I've personally been using user0@greencommons.org and thecommons to login since I started working on the app. Should we create a separate one and use create_list for the other users?